mev-commit-cli.sh

```diff
@@ -15,7 +15,7 @@ MEV_COMMIT_PATH="$PRIMEV_DIR/$MEV_COMMIT_REPO_NAME"
 ORACLE_PATH="$PRIMEV_DIR/$ORACLE_REPO_NAME"
 
 DOCKER_NETWORK_NAME="primev_net"
-MEV_COMMIT_BRANCH="main"
+MEV_COMMIT_BRANCH="ckartik/switch-to-goerli"
```

Comment:
if we want to switch to goerli then the main branch should also have the related changes if there are any